hedera-node/hedera-network-admin-service-impl/src/main/java/module-info.java

```diff
@@ -14,7 +14,7 @@
     requires com.hedera.node.app.service.file;
     requires com.hedera.node.app.service.token;
     requires com.google.common;
-    requires com.swirlds.config;
+    requires com.swirlds.config.api;
```

Comment:
The change is because the project in Gradle is named `swirlds-config-api` as is the folder of the project. This aligns the module name to that. We can change it back (in a follow up "naming cleanup"?) and adjust the Gradle project (and folder name) instead.